### PR TITLE
updating fontawesome assets from v5.15.3 to v6.5.2

### PR DESCRIPTION
**Pull request by jkellz-dev**
To get some of the newer icons for the brands, I pulled the updated free FA icons. I'm not certain all of the files are necessary, but I put everything there just in case.

### CODE REVIEW

**Comment by fncnt** [comment]
Thank you! Looks mostly good to me after checking the `sha512` sums of the files against the original archive from fontawesome.
It appears that you took `sprites/*.svg` and renamed them to match the `webfonts/*.svg` files I used from fontawesome `5.15.3`. 
In `6.5.2`, `webfonts/` doesn't contain any `.svg` files anymore, so I'd prefer if you could remove them in this PR too.

I'm happy to merge your PR afterwards.

File: static/webfonts/fa-brands-400.svg
Comment (fncnt) [inline static/webfonts/fa-brands-400.svg:1]: This file doesn't exist anymore in fontawesome `6.5.2`. Please delete instead.

File: static/webfonts/fa-regular-400.svg
Comment (fncnt) [inline static/webfonts/fa-regular-400.svg:1]: This file doesn't exist anymore in fontawesome `6.5.2`. Please delete instead.

File: static/webfonts/fa-solid-900.svg
Comment (fncnt) [inline static/webfonts/fa-solid-900.svg:1]: This file doesn't exist anymore in fontawesome `6.5.2`. Please delete instead.

**Review by fncnt — CHANGES_REQUESTED** [review]

**Comment by jkellz-dev** [comment]
Oh, yeah, I totally made an assumption without actually digging into it at all. My bad. I'll update right away.

File: static/webfonts/fa-brands-400.svg
Comment (jkellz-dev) [reply static/webfonts/fa-brands-400.svg:1]: removed

**Comment by fncnt** [comment]
Thanks, merged!

**Review by fncnt — APPROVED** [review]

**Pull request merged**